internal/app/osd/internal/reg/reg_test.go

```diff
@@ -0,0 +1,17 @@
+package reg
+
+import (
+	"net"
+	"testing"
+
+	"golang.org/x/sys/unix"
+)
+
+func TestToCIDR(t *testing.T) {
```

Comment:
nit, we've been standardizing on using testify for our tests. ex, https://github.com/talos-systems/talos/blob/master/internal/app/init/pkg/system/system_test.go

Reply:
Thanks; I should have checked.